### DISCUSSION TITLE
can't stop email notifications

### DISCUSSION THREAD

**Discussion by arpad9** [community/community#22932] [New to GitHub]
<p>This seems really stupid.  I can’t seem to stop email notifications for repos.  Pull requests comments, commits, etc…   I’ve checked my Notification settings and Email is unchecked.  I’ve now even Unwatched all repos, not my preference because I’d still like web notifications.  But the emails keep coming.  Floods my Inbox.  Too much noise.  Absolutely unusable.</p>
<p>I must be missing something idiotic.  Help?</p>

**Comment by ghost** [comment, marked as answer]
<p>Hi <a class="mention" href="/arpad9">@arpad9</a>,</p>
<p>It sounds like you might be subscribed to certain notifications on repos and in conversations which you can unsubscribe from using this help guide: <a href="https://help.github.com/articles/subscribing-to-and-unsubscribing-from-notifications/" rel="nofollow noopener">https://help.github.com/articles/subscribing-to-and-unsubscribing-from-notifications/</a></p>
<p>If that doesn’t work, you’ll probably want to contact support for help at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a></p>

**Comment by arpad9** [comment]
<p>Oh, I figured it out.  Always look at mail headers.  It’s as a part of an email list I’m also on.</p>
<p>Thanks for responding.</p>
